Source/WebCore/PAL/pal/text/TextEncoding.cpp

```diff
@@ -62,15 +62,15 @@ TextEncoding::TextEncoding(const String& name)
 
 String TextEncoding::decode(const char* data, size_t length, bool stopOnError, bool& sawError) const
 {
-    if (!m_name)
+    if (m_name.isEmpty())
```

Comment:
This is a semantic change. The old code checks for null and the new code checks for empty. Just want to make sure you did that intentionally.

Reply:
Ouch, this wasn't at all intentional. I am glad you caught it, thank you.